src/main/java/com/laborete/LaboreteAPI/profile/services/UsersService.java

```diff
@@ -9,6 +10,6 @@
 
 public interface UsersService {
     UserEntity getUserById(UUID id);
-    UserEntity createUser(UserEntity user);
+    UserEntity createUser(UserDTO user);
```

Comment:
For me it looks strange that the method take DTO as a parameter and returns entity. It would be better to return the DTO and do the mapping inside. 

UserEntity is a Hibernate entity. The service itself should not be tied to entities because it should be reusable. What if I want to implement this service and store users not in the database, but in any other storage?